docs/firstobject.md

```diff
@@ -76,78 +76,84 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
+**Note:** For now, let's leave the state of this Digital Object as `Draft`.
 
-<details><summary>Why does this look different than Repository X?</summary>
+<details><summary>"Why does this look different than Repository X?"</summary>
 <div>
 
-We assume you come from a world where repositories define different Content types and the shape, the fields and values (Schema) are fixed and set by someone else or at least quite complicated to configure. This is where Archipelago differs and starts to propose its own style. You noticed that there is a single Content Type named `Digital Object` and you have here a single Web Form. So how does this allows you to have Images, sequences, Videos, audio, 3D images, etc? 
+We assume you come from a world where repositories define different Content types and the shape, the fields and values (Schema) are fixed and set by someone else or at least quite complicated to configure. This is where Archipelago differs and starts to propose its own style. You noticed that there is a single Content Type named `Digital Object` and you have here a single Web Form. So how does this allow you to have images, sequences, videos, audio, 3D images, etc?
 
-There are many ways of answering that, archipelago works under the idea of an (or many) Open Schema(s), and that notion permeates the whole environment. Practical answers, simplest way to explain this based on this demo is:
+There are many ways of answering that, Archipelago works under the idea of an (or many) Open Schema(s), and that notion permeates the whole environment. Practical answer and simplest way to explain based on this demo is:
 
-- The `Digital Object` is a generic container for any shape of metadata. Metadata is generated either via this Webform based widget, manually (no worries, power-user need only) or via APIs and can take because of that any shape to express your needs of Digital Objects. You really don't need many types of Digital Objects (trust us!). If you ever need them, those are easy to setup.
-- The Strawberry field Field Widget allows you to attach any `Webform`, built using the [`Webform Module`](https://www.drupal.org/docs/8/modules/webform) and Webforms can be setup in almost infinite ways. Any field, combo, style can be used. Multi Step, single step. We made sure they always only touch/modify data they know how to touch, so even a single input element webform would ensure any previous metadata to persist even if not readable by itself (See the potential?). And Each Webform can be also quite smart!
+- The `Digital Object` is a generic container for any shape of metadata. Metadata is generated either via this Webform-based widget you're currently using, manually (power-user need only) or via APIs. Because of this, Metadata can take any shape to express your needs of Digital Objects and therefore we do not recommend making multiple Digital Object types. However, if you ever do need more Digital Object types, the option is available.
```

Comment:
I'm really not convinced about the wording here. The `we don't recommend` and then the `option is available` somehow differs from the original tone. I will suggest we use your new wording but rethink this on beta3.